Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -428,7 +428,9 @@ WarpX::shiftMF (amrex::MultiFab& mf, const amrex::Geometry& geom,
             dstfab(i,j,k,n) = srcfab(i+shift.x,j+shift.y,k+shift.z,n);
         })
 
-        if (cost && WarpX::load_balance_costs_update_algo == LoadBalanceCostsUpdateAlgo::Timers)
+        if (cost &&
+            mfi.index() < cost->size() && //Workaround to fix invalid memory accesses when mf is a PML
```

Comment:
attn @atmyers @WeiqunZhang : PMLs have a different domain decomposition than regular MFs. How can we handle costs for them (see: `&(*cost)[mfi.index()]` below)?

Reply:
> This function could simply check whether it's working on PML grids and skip updating the costs if so.
> ... In summary, I think it's probably fine just to skip updating the costs for PML grids.

I think that would be a good solution for now.

Reply:
ping @lucafedeli88 :)

Reply:
Sure! Sorry... I'll try to do that now...

Reply:
@ax3l , would this be OK as a temporary workaround ?

Reply:
LGTM - @atmyers do you finalize the review and merge? :)